protocol/message_processor.go

```diff
@@ -171,13 +171,41 @@ func (p *messageProcessor) sendPrivate(
 	return messageID, nil
 }
 
-func (p *messageProcessor) SendMembershipUpdate(
+// sendPairInstallation sends data to the recipients, using DH
+func (p *messageProcessor) SendPairInstallation(
 	ctx context.Context,
-	recipients []*ecdsa.PublicKey,
+	recipient *ecdsa.PublicKey,
+	data []byte,
+	messageType protobuf.ApplicationMetadataMessage_Type,
+) ([]byte, error) {
+	p.logger.Debug("sending private message", zap.Binary("recipient", crypto.FromECDSAPub(recipient)))
+
+	wrappedMessage, err := p.wrapMessageV1(data, messageType)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to wrap message")
+	}
+
+	messageID := v1protocol.MessageID(&p.identity.PublicKey, wrappedMessage)
```

Comment:
Nitpick: It's used at the bottom so we can push it just before `p.transport.Track(...)`.